code/network/lib/Loot/Network/ZMQ/Client.hs

```diff
@@ -377,14 +413,15 @@ reconnectPeers :: MonadIO m => ZTNetCliEnv -> Set ZTNodeId -> m ()
 reconnectPeers ZTNetCliEnv{..} nIds = liftIO $ do
     ztLog ztCliLogging Warning $ "Reconnecting peers: " <> show nIds
 
+
     forM_ nIds $ \nId -> do
         Z.disconnect ztCliBack (ztNodeIdRouter nId)
         Z.connect ztCliBack (ztNodeIdRouter nId)
         Z.disconnect ztCliSub (ztNodeIdPub nId)
         Z.connect ztCliSub (ztNodeIdPub nId)
     curTime <- getCurrentTimeMs
     atomically $ do
-        let mulTwoMaybe x | x >= hbIntervalMax = x
+        let mulTwoMaybe x | x >= zsHBIntervalMax ztCliSettings = x
                           | otherwise = 2 * x
```

Comment:
Note that this function allows twice as higher polling intervals than `zsHBIntervalMax`, and since `zsHBIntervalMax` is a public parameter now, I think logic should correspond to its name.

Reply:
It's not a bug, it's just how it was designed (I copied everything from the zmq guide). But what you say makes sense -- I'll double if it's less than max interval, and will set to max interval otherwise.